### PR TITLE
adds new element to html body for greeting message to be displayed

### PR DESCRIPTION
**Pull request by hajiler**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,7 +26,13 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
-
+git
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/script.js]: git this out of here :)

This still happens to me when I forget to switch tabs and think I'm typing into a terminal but am still in my text editor haha.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,7 +26,13 @@ function addRandomGreeting() {
...
   greetingContainer.innerText = greeting;
 }
-
+git 
 function openLink(link){
     window.open(link);
 }
+
+function getHello() {
+    fetch('/data').then(response => response.text()).then((greeting) => {
```
Comment (jessrenteria) [inline portfolio/src/main/webapp/script.js]: Nit: 2 space indent.

It looks like Cloud Shell Editor uses 4 spaces by default, but you can change this to 2 in your preferences:

File > Settings > Preferences > Editor > tabSize

**Review by jessrenteria — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (hajiler) [reply portfolio/src/main/webapp/script.js]: Oh wow I gotta watch out for that, I fixed it in the next commit.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (hajiler) [reply portfolio/src/main/webapp/script.js]: Thanks, its a habit for me to use tab, that will definitely help with my spacing errors.

**Review by jessrenteria — APPROVED** [review]

**Pull request merged**